backend/typescript/package.json

```diff
@@ -61,5 +67,9 @@
     "nodemon": "^2.0.7",
     "prettier": "^2.2.1",
     "typescript": "^4.1.5"
+  },
+  "resolutions": {
```

Comment:
This is because of an issue with apollo-server-express using an older version of graphql-upload - read more here https://github.com/apollographql/apollo-server/issues/3508